include/CoreLocation/CLGeocoder.h

```diff
@@ -40,4 +40,10 @@ CORELOCATION_EXPORT_CLASS
            completionHandler:(CLGeocodeCompletionHandler)completionHandler;
 - (void)cancelGeocode STUB_METHOD;
 @property (readonly, getter=isGeocoding, nonatomic) BOOL geocoding;
-@end
\ No newline at end of file
+@end
+
+@interface CLGeocoder (WinObjC)
+// WINOBJC extension method to set underlying Bing service token, required to use CLGeocoder APIs
+// See https://docs.microsoft.com/en-us/windows/uwp/maps-and-location/geocoding for more info
++ (void)_setServiceToken:(NSString*)token;
```

Comment:
nit: underscores are for private things; `setWindowsServiceToken:` might be a good name?